lib/UserAgentApplication.ts

```diff
@@ -50,19 +50,6 @@ namespace Msal {
         }
```

Comment:
@austindyoung Adding an options object will lead to a breaking change in the way people are calling our api's using our samples.